sphinx/source/docs/user_guide/examples/extensions_putting_together_ts.py

```diff
@@ -66,22 +66,20 @@
   }
 
   static init_Custom(): void {
-    // If there is an associated view, this is typically boilerplate.
+    // If there is an associated view, it's typically a boilerplate.
```

Comment:
I don't like this new wording, it's the code for the view that is typically boilerplate, and I don't think this conveys that exactly